packages/react-components/src/lib/cesium-map/data-sources/drawings.data-source.stories.tsx

```diff
@@ -84,8 +86,62 @@ export const Drawings: Story = (args) => {
       >
         Box
       </button>
+      <button
+        style={{ position: 'fixed', top: '80px', left: '20px', zIndex: 1 }}
+        onClick={(): void => {
+          setIsDrawing(false);
+          setDrawPrimitive({
+            type: DrawType.UNKNOWN,
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+            handler: (drawing: IDrawingEvent) => {},
```

Comment:
do you have to supply handler for draw end? is so, why?
what will be in the drawing object in that case?

Reply:
Actually, no need to supply, it's cleaned up internally